dotcom-rendering/src/web/server/index.allEditorialNewslettersPage.web.ts

```diff
@@ -19,11 +17,8 @@ export const handleAllEditorialNewslettersPage: RequestHandler = (
 	{ body },
 	res,
 ) => {
-	try {
-		const newslettersPage = enhanceAllEditorialNewslettersPage(body);
-		const html = renderEditorialNewslettersPage({ newslettersPage });
-		res.status(200).send(html);
-	} catch (e) {
-		res.status(500).send(`<pre>${getStack(e)}</pre>`);
```

Comment:
Catching the error here stops our ErrorRequestHandler from handling it.

The ErrorRequestHandler is pretty much doing exactly the same thing but it has a few extra bells and whistles for logging and handling different error types.